modules/oe_translation_cdt/src/TranslationRequestCdt.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_translation_cdt;
+
+use Drupal\oe_translation\Entity\TranslationRequest;
+use Drupal\oe_translation_remote\RemoteTranslationRequestEntityTrait;
+use Drupal\oe_translation_remote\TranslationRequestRemoteInterface;
+
+/**
+ * A CDT bundle class for oe_translation_request entities.
+ */
+final class TranslationRequestCdt extends TranslationRequest implements TranslationRequestCdtInterface {
+  use RemoteTranslationRequestEntityTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCdtId(): ?string {
+    return $this->get('cdt_id')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setCdtId(string $value): TranslationRequestCdtInterface {
+    $this->set('cdt_id', $value);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setRequestStatusFromCdt(string $cdt_status): TranslationRequestCdtInterface {
+    $target_status = match($cdt_status) {
+      'COMP' => TranslationRequestRemoteInterface::STATUS_REQUEST_TRANSLATED,
```

Comment:
these constants, COMP, CANC, CMP, should be defined in the cdt client and imported here
From the docs its a bit dubious, the callback definition can send CMP or CNC, but we can get back COMP

Reply:
yes, I forgot